apps/ocp-plugin/console-extensions.json

```diff
@@ -141,6 +151,14 @@
       "component": { "$codeRef": "EnrollmentRequestDetailsPage" }
     }
   },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/edge/overview"],
+      "component": { "$codeRef": "OverviewTab" }
```

Comment:
Instead of OverviewTab we should create a new `codeRef` for the whole [OverviewPage](https://github.com/flightctl/flightctl-ui/blob/main/libs/ui-components/src/components/OverviewPage/OverviewPage.tsx). OverviewTab does not contain Page title.